packages/app/src/app/pages/Sandbox/Editor/Content/Tabs/Tab/Tab.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from 'react';
+import ContextMenu from 'app/components/ContextMenu';
+import { Container, TabTitle, StyledCloseIcon } from './elements';
+
+export const Tab = ({
+  active,
+  dirty,
+  isOver,
+  position,
+  tabCount,
+  closeTab,
+  onClick,
+  onDoubleClick,
+  children,
+  title,
+  items,
+}) => {
+  const [isHovering, setHovering] = useState(false);
+
+  const handleMouseEnter = () => setHovering(true);
+
+  const handleMouseLeave = () => setHovering(false);
+
+  const handleCloseTab = e => {
+    e.preventDefault();
+    e.stopPropagation();
+
+    if (closeTab) {
```

Comment:
@Saeris If you give `closeTab` a default value to `noop` (ie `() => undefined`), you don't need this conditional